types/fulu/execution_payload.yaml

```diff
@@ -0,0 +1,80 @@
+Fulu:
+  ExecutionPayloadCommon:
+    # An abstract object to collect the common fields between the ExecutionPayload and the ExecutionPayloadHeader objects.
+    type: object
+    # TODO: update link once confirmed.
+    description: "The [`ExecutionPayload`](https://github.com/ethereum/consensus-specs/blob/v1.5.0-beta.2/specs/fulu/beacon-chain.md#executionpayload) object from the CL Fulu spec."
+    required:
+      [
+        parent_hash,
+        fee_recipient,
+        state_root,
+        receipts_root,
+        logs_bloom,
+        prev_randao,
+        block_number,
+        gas_limit,
+        gas_used,
+        timestamp,
+        extra_data,
+        base_fee_per_gas,
+        blob_gas_used,
+        excess_blob_gas,
+        block_hash,
+      ]
+    properties:
+      parent_hash:
+        $ref: "../primitive.yaml#/Root"
+      fee_recipient:
+        $ref: "../primitive.yaml#/ExecutionAddress"
+      state_root:
+        $ref: "../primitive.yaml#/Root"
+      receipts_root:
+        $ref: "../primitive.yaml#/Root"
+      logs_bloom:
+        $ref: "../primitive.yaml#/LogsBloom"
+      prev_randao:
+        $ref: "../primitive.yaml#/Bytes32"
+      block_number:
+        $ref: "../primitive.yaml#/Uint64"
+      gas_limit:
+        $ref: "../primitive.yaml#/Uint64"
+      gas_used:
+        $ref: "../primitive.yaml#/Uint64"
+      timestamp:
+        $ref: "../primitive.yaml#/Uint64"
+      extra_data:
+        $ref: "../primitive.yaml#/ExtraData"
+      base_fee_per_gas:
+        $ref: "../primitive.yaml#/Uint256"
+      blob_gas_used:
+        $ref: "../primitive.yaml#/Uint256"
+      excess_blob_gas:
+        $ref: "../primitive.yaml#/Uint256"
+      block_hash:
+        $ref: "../primitive.yaml#/Root"
+
+  ExecutionPayload:
+    allOf:
+      - $ref: "#/Fulu/ExecutionPayloadCommon"
+      - type: object
+        required: [transactions, withdrawals, proof_version]
+        properties:
+          transactions:
+            $ref: "../bellatrix/transactions.yaml#/Bellatrix/Transactions"
+          withdrawals:
+            $ref: "../capella/withdrawals.yaml#/Capella/Withdrawals"
+          proof_version:
+            $ref: "../primitive.yaml#/Uint8"
+
+  ExecutionPayloadHeader:
+    description: "The [`ExecutionPayloadHeader`](https://github.com/ethereum/consensus-specs/blob/v1.4.0/specs/deneb/beacon-chain.md#executionpayloadheader) object from the CL Deneb spec."
```

Comment:
```suggestion
    # TODO: update link once confirmed.
    description: "The [`ExecutionPayloadHeader`](https://github.com/ethereum/consensus-specs/blob/v1.4.0/specs/deneb/beacon-chain.md#executionpayloadheader) object from the CL Fulu spec."
```

Reply:
no longer needed as there is no change to `ExecutionPayload` now